### PR TITLE
Remove need for separate ReSpec clone for Arazzo

### PR DESCRIPTION
**Pull request by ralfhandl**
Use the same OAI fork of ReSpec as the OpenAPI specification and set section headline for "abstract" in the HTML document instead of tweaking the default in the `respec-xxx.js`

### CODE REVIEW

**Comment by ralfhandl** [comment]
@frankkilcommins Please review

**Comment by frankkilcommins** [comment]
@ralfhandl can you share the related OpenAPI-Specification/gh-pages commit?

**Comment by ralfhandl** [comment]
Not yet merged, PR https://github.com/OAI/OpenAPI-Specification/pull/3943 is still waiting for @darrelmiller's approval.

**Review by frankkilcommins — APPROVED** [review]
LGTM - we'll merge after the https://github.com/OAI/OpenAPI-Specification/pull/3943 is merged.

**Comment by ralfhandl** [comment]
OAS is moving to latest unmodified ReSpec with static HTmL pages that so not need a `respec-*.js`.

**Pull request closed without merging**
